plugins/language_gmi.lua

```diff
@@ -0,0 +1,22 @@
+-- mod-version:2 -- lite-xl 2.0
+local syntax = require "core.syntax"
+
+
+
+syntax.add {
+  name = "Gemtext",
+  files = { "%.gmi$" },
+  patterns = {
+    { pattern = { "```", "```" },           type = "string"   },
+    { pattern = "#.-",                      type = "keyword"  },
+    { pattern = "%*.*",                     type = "keyword2" },
+    { pattern = "=>",                       type = "function" },    
+    { pattern = "!?%[.-%]%(.-%)",           type = "function" },
```

Comment:
Does GemText support this style of links?

Reply:
i forgot to remove line 14 (gemtext doesn't support that style of links), i'll remove that asap.